packages/flex-plugin-scripts/src/scripts/start.ts

```diff
@@ -38,6 +38,12 @@ interface Packages {
   pkg: Record<string, string>;
 }
 
+export interface PluginConfig {
```

Comment:
Delete this - you've already defined this in the devServer file use that one instead.